config-service/build.gradle.kts

```diff
@@ -73,10 +73,7 @@ dependencies {
   integrationTestImplementation("org.hypertrace.core.serviceframework:integrationtest-service-framework:0.1.18")
   integrationTestImplementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.3.3")
   constraints {
-    runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final") {
-      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1070799")
-    }
-    runtimeOnly("io.netty:netty-handler-proxy:4.1.59.Final") {
+    runtimeOnly("io.netty:netty-handler-proxy:4.1.60.Final") {
       because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1070799")
```

Comment:
nit: could you update the reason also?

Reply:
Isn't the reason the snyk link? Didn't follow you 🤔

Reply:
yes, the snyk link is updated now - `https://snyk.io/vuln/SNYK-JAVA-IONETTY-1083991`. That link points to an older vulnerability.

Reply:
Not really a blocker I guess.

Reply:
Oh I see. Done